content/_data/authors/dkjellin.adoc

```diff
@@ -0,0 +1,6 @@
+---
+name: Daniel Kjellin
+github: "dkjellin"
+---
+
+Short description to be written by Daniel. 
```

Comment:
@dkjellin can you please add your bio here? Thanks.

Reply:
```suggestion
Daniel has been with link:https://www.atlassian.com[Atlassian] over ten years working on a variety of different products. Lately he has lead the work to write a new and improved integration between link:https://www.atlassian.com/software/bitbucket/enterprise/data-center[Bitbucket Server] and Jenkins. 
```